back/app/uploaders/header_bg_uploader.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# Uploads a header background image in large, medium, small sizes and stores them in a directory based on the
+# AppConfiguration id and model id.
+class HeaderBgUploader < BaseImageUploader
+  def store_dir
+    File.join('uploads', AppConfiguration.instance.id, 'header-background', model.id)
```

Comment:
See https://github.com/citizenlabdotco/citizenlab-ee/blob/CL-909_add_banner_image_to_home_page/back/engines/ee/multi_tenancy/app/uploaders/multi_tenancy/patches/app_header_bg_uploader.rb#L8-L10. Basically the same implementation, but based on the ID of `AppConfiguration` instead of `Tenant`.